src/components/scenes/TransactionListScene.js

```diff
@@ -87,17 +115,31 @@ class TransactionListComponent extends React.Component<Props, State> {
     return this.props.loading ? <ActivityIndicator style={{ flex: 1, alignSelf: 'center' }} size="large" /> : <TransactionListHeader />
   }
 
-  renderTx = (transactionListItem: FlatList<TransactionListTx>) => {
-    const { transactions } = this.props
-    const transaction = transactionListItem.item
-    const isHeader = transaction.key === 0 || transaction.dateString !== transactions[transaction.key - 1].dateString
-    return <TransactionRow transaction={transaction} isHeader={isHeader} />
+  renderSectionHeader = (section: { section: Section }) => {
+    const styles = getStyles(this.props.theme)
+    return (
+      <View style={styles.headerContainer}>
+        <EdgeText style={styles.formattedDate}>{section.section.title}</EdgeText>
+      </View>
+    )
+  }
+
+  renderTransaction = (transaction: SectionList<TransactionListTx>) => {
+    return <TransactionRow transaction={transaction.item} />
   }
 }
 
 const getStyles = cacheStyles((theme: Theme) => ({
-  transactionsScrollWrap: {
-    flex: 1
+  headerContainer: {
+    // marginTop: theme.rem(0.5),
+    marginBottom: theme.rem(0.125),
+    backgroundColor: theme.tileBackground,
```

Comment:
While I know this is transparent, this should be a sectionHeaderBackgroundColor to potentially distinguish it from the transaction row background

Reply:
No, this should be gradient, see next comment on the 2.0 design